Sources/Orbit/Components/InputField.swift

```diff
@@ -160,15 +195,64 @@ public extension InputField {
     }
 }
 
+private struct SecureTextField: UIViewRepresentable {
+    typealias UIViewType = UITextField
+
+    @Binding var text: String
+    @Binding var isSecured: Bool
+
+    func makeUIView(context: Context) -> UITextField {
+        let textFied = UITextField()
```

Comment:
The textField is not using Orbit font. Size, font and color should match the normal input field.